packages/config/src/layer2s/arbitrum.ts

```diff
@@ -53,6 +53,7 @@ export const arbitrum: Layer2 = {
     activityDataSource: 'Blockchain RPC',
   },
   config: {
+    associatedTokens: ['ARB'],
     escrows: [
       {
         address: EthereumAddress('0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a'),
```

Comment:
ARB token on L1 is a wrapper for ARB token on L2, no ? So you will be counting this token twice

Reply:
No we don't, as the token is locked on L2 and minted on L1